src/GitHub.App/Services/PullRequestService.cs

```diff
@@ -328,12 +328,14 @@ public IObservable<Unit> SwitchToBranch(ILocalRepositoryModel repository, IPullR
             });
         }
 
-        public Encoding GetEncoding(string path)
+        public Encoding GetEncoding(ILocalRepositoryModel repository, string relativePath)
         {
-            if (File.Exists(path))
+            var fullPath = Path.Combine(repository.LocalPath, relativePath);
+
+            if (File.Exists(fullPath))
```

Comment:
We probably shouldn't silently return `Encoding.Default` if the file doesn't exist. I'm guessing this is what allowed the isssue to slip through in the first place.

Is it ever normal for the file not to exist? Do you think we should throw a file not found exception or something?